src/annotation.ts

```diff
@@ -50,6 +50,7 @@ export class Annotation {
       }
     }
 
+    // TODO: why not use this.run?
```

Comment:
Let's address that (after double checking that it doesn't break anything) immediately instead of just leaving a TODO note 🙂